### PR TITLE
✨ Bump CPI to v1.31.0-beta.0

### PR DESCRIPTION
**Pull request by sbueringer**
Signed-off-by: Stefan Büringer buringerst@vmware.com

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Review by chrischdi — COMMENTED** [review]
/lgtm
/approve

**Review by fabriziopandini — COMMENTED** [review]
/lgtm
/approve

File: packaging/flavorgen/cloudprovider/cpi/cpi.yaml [hunk 1]
```diff
@@ -155,7 +155,7 @@ spec:
       priorityClassName: system-node-critical
       containers:
       - name: vsphere-cpi
-        image: gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.30.0-rc.0
+        image: registry.k8s.io/cloud-pv-vsphere/cloud-provider-vsphere:v1.30.0-rc.0
```
Comment (sbueringer) [inline packaging/flavorgen/cloudprovider/cpi/cpi.yaml:158]: We should use the new community-owned registry

**Review by chrischdi — COMMENTED** [review]
/lgtm
/approve

**Review by fabriziopandini — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**